File: test/fixtures/output/ocaml/cohttp/application-form-encoded.ml
open Cohttp_lwt_unix
open Cohttp
open Lwt

let uri = Uri.of_string "http://mockbin.com/har" in
let headers = Header.init ()
  |> fun h -> Header.add h "content-type" "application/x-www-form-urlencoded"
in
let body = "foo=bar&hello=world" in

Client.call ~headers ~body `POST uri
>>= fun (res, body_stream) ->
  (* Do stuff with the result *)

File: src/targets/php/guzzle/fixtures/application-form-encoded.php
<?php

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'http://mockbin.com/har', [
  'form_params' => [
    'foo' => 'bar',
    'hello' => 'world'
  ],
  'headers' => [
    'content-type' => 'application/x-www-form-urlencoded',
  ],
]);

echo $response->getBody();